src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final String DEF_COLOR = "WHITE";
+    private static final int MAX_NUMBER = 10;
+    private static final int MIN_NUMBER = 1;
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getDefaultFigure() {
+        return new Circle(DEF_COLOR, MAX_NUMBER);
+    }
+
+    public Figure getRandomFigure() {
+        int randomNumberFirst = random.nextInt(MIN_NUMBER, MAX_NUMBER);
```

Comment:
create separate private method for `random.nextInt(MIN_NUMBER, MAX_NUMBER)` and use it for creating each type of figure